query/filter.go

```diff
@@ -236,6 +240,79 @@ func (e And) String() string {
 	return fmt.Sprintf(`{"$and":[%s]}`, strings.Join(subS, ", "))
 }
 
+type In struct {
+	Min    []byte
+	Max    []byte
+	Values map[string]struct{}
```

Comment:
Let's make min/max/values private, because in public we can't guarantee the consistency between values and min/max